src/Postmark/PostmarkClient.php

```diff
@@ -42,6 +42,7 @@ function __construct($serverToken, $timeout = 60) {
 	 * @param  array $metadata  Add metadata to the message. The metadata is an associative array, and values will be evaluated as strings by Postmark.
 	 * @param  array $messageStream  The message stream used to send this message. If not provided, the default transactional stream "outbound" will be used.
 	 * @return DynamicResponseModel
+	 * @throws Models\PostmarkException
```

Comment:
```suggestion
         * @throws Models\PostmarkException
         *
	 * @return DynamicResponseModel
```

Looking at the [phpDocumentor example for throws](https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/throws.html) do we want the `return` tag to be last?